### DISCUSSION TITLE
Question: when does the website get rebuilt after a new release?

### DISCUSSION THREAD

**Discussion by dr-shorthair** [qudt/qudt-public-repo#301] [Q&A] [answered]

**Comment by steveraysteveray** [comment, marked as answer]
The website is published roughly once per month, but normally it is done whenever the GitHub repo is relatively stable (i.e. no major branches about to be merged or outstanding). New releases are normally done a little less often, whenever there is significant new functionality or population. And as you know, the GitHub repo itself evolves continuously.

**Reply by VladimirAlexiev** [reply]
https://github.com/qudt/qudt-public-repo/issues/375 is a wider question
